src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnection.cs

```diff
@@ -1138,7 +1203,6 @@ private void ReturnConnectionToPoolCore()
                 try
                 {
                     // Null out the associated request before the connection is potentially reused by another.
-                    _currentRequest = null;
```

Comment:
Why was this removed?  If it's intentional, can we update the comment?

Reply:
Because we already did it:
https://github.com/stephentoub/corefx/blob/a137bbf42598202f999ead31d92754613145c849/src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnection.cs#L1149

I'll remove the stray comment.